stock_landed_costs_average/model/stock_landed_costs.py

```diff
@@ -587,7 +588,13 @@ def compute_landed_cost(self, cr, uid, ids, context=None):
                         else:
                             value = (line.price_unit / total_line)
 
-                        if digits:
+                        # When the valuation is the last record, assign
+                        # line.price_unit less the accumulated value in
+                        # value_split
+                        if digits and \
+                                valuation == cost_adjust_lines[-1]:
```

Comment:
I do not understand this `cost_adjust_lines` is using by `for` but in this validation
`cost_adjust_lines[-1]` always is returning the last position , why?